pkg/registry/servicecatalog/broker/storage.go

```diff
@@ -34,7 +34,7 @@ import (
 	"k8s.io/apiserver/pkg/registry/generic/registry"
 	"k8s.io/apiserver/pkg/registry/rest"
 	"k8s.io/apiserver/pkg/storage"
-	"k8s.io/client-go/pkg/api"
+	coreapi "k8s.io/client-go/pkg/api"
```

Comment:
why this change? It does not look like we imported our 'api' here.

Reply:
@MHBauer 2 reasons:
1) I moved the scheme to our own `pkg/api` package and refer to it simply as `api` everywhere.
2) in 1.8 the package is moved to a separate repo https://github.com/kubernetes/api/tree/master/core/v1
see `core/v1`, hence the core alias prefix